fvm/Cargo.toml

```diff
@@ -58,3 +58,7 @@ m2-native = []
 upgrade-actor = []
 gas_calibration = []
 nv23-dev = []
+# Use this feature to remove `verify_signature` syscall that is supposed to be removed by FIP-0079,
+# The current implementation keeps it by default for backward compatibility reason.
+# See <https://github.com/filecoin-project/ref-fvm/issues/2001>
+no-verify-signature = []
```

Comment:
Let's add this as a default feature. I.e.:

1. Add a `verify-signature` syscall feature.
2. Add that feature to the default feature's array.

Reply:
Fixed.